google/cloud/bigquery/client.py

```diff
@@ -2850,7 +2850,7 @@ def _do_resumable_upload(
         )
 
         while not upload.finished:
-            response = upload.transmit_next_chunk(transport)
+            response = upload.transmit_next_chunk(transport, timeout=timeout)
```

Comment:
FWIW, the [`timeout` argument](https://github.com/googleapis/google-resumable-media-python/blob/d0827725085ab064169cda004a4fde65e92813f0/google/resumable_media/_upload.py#L753-L759) can actually be a 2-tuple as well:

```
            timeout (Optional[Union[float, Tuple[float, float]]]):
                The number of seconds to wait for the server response.
                Depending on the retry strategy, a request may be repeated
                several times using the same timeout each time.

                Can also be passed as a tuple (connect_timeout, read_timeout).
                See :meth:`requests.Session.request` documentation for details.
```

Maybe we should expose that in the docstring for this method, and its one caller (`Client.load_table_from_file`).

Reply:
Makes sense, done.